Auth0/Authentication.swift

```diff
@@ -774,15 +774,15 @@ public extension Authentication {
      ```
 
      - parameter credentials: the client's credentials obtained on auth
-     - parameter scope: scopes to request for the new tokens. By default is nil which will ask for the same ones requested during Auth.
+     - parameter scope: scopes to request for the new tokens. By default is nil which will ask for the same ones requested during Auth
+     - parameter callback: callback with the user's credentials or the cause of the error.
      - important: This method only works for a refresh token obtained after auth with OAuth 2.0 API Authorization.
-     - returns: a request that will either yield an error or Auth0 user's credentials
      */
     func renewExpired(_ credentials: Credentials, scope: String? = nil, callback: @escaping (Error?, Credentials?) -> Void) {
-        guard
-            let refreshToken = credentials.refreshToken,
-            let expiresIn = credentials.expiresIn, expiresIn < Date(timeIntervalSinceNow: -60)
-            else { return callback(AuthenticationError(string: "missing refresh_token or have not expired", statusCode: 400), nil) }
+        guard let refreshToken = credentials.refreshToken else {
+            return callback(AuthenticationError(string: "missing refresh_token", statusCode: 400), nil)
+        }
+        guard let expiresIn = credentials.expiresIn, expiresIn < Date(timeIntervalSinceNow: -60) else { return callback(nil, credentials) }
```

Comment:
I added the 60 although it should perhaps be longer based like 5 minutes.  As it would be if a token was not renewed as it was close to expiry and then immediately expires when the user goes to use it a few seconds later.
I've not looked at this in the JWT Libs, although I feel if a user's clock is wrong this isn't really our issue.